README.md

```diff
@@ -203,6 +207,15 @@ Configure the action by customizing the following parameters based on your needs
     with:
       verbose-logging: true
     ```
+    
+- **structured-output** (optional, `default: false`)
```

Comment:
Our default setting should provide the information-rich and easily read output.
Maybe the structured output should be the default here.

Reply:
When we were discussing the `project-state-mining` feature, we came to a closure, that in default we want to hand over the BASE output. Meaning no project state data, no verbose logging. And in my opinion also unstructured output, this should be probably the same thing. It would look better, but it goes against our thoughts. What do you thing?

Reply:
You are right. I am influenced by my GH Action where the output should look nice at in default.